scripts/sct_qc.py

```diff
@@ -31,7 +31,7 @@ def get_parser():
                         help='SCT function associated with the QC report to generate',
                         choices=('sct_propseg', 'sct_deepseg_sc', 'sct_deepseg_gm', 'sct_register_multimodal',
                                  'sct_register_to_template', 'sct_warp_template', 'sct_label_vertebrae',
-                                 'sct_detect_pmj'),
+                                 'sct_detect_pmj', 'sct_label_utils'),
                         required=True)
     parser.add_argument('-s',
                         metavar='SEG',
```

Comment:
we probably want to change "help='Input segmentation'," for "help='Input segmentation or label',"